cfme/tests/control/test_alerts.py

```diff
@@ -185,6 +185,15 @@ def vm_crud(provider, vm_name, full_template):
     return VM.factory(vm_name, provider, template_name=full_template["name"])
 
 
+@pytest.yield_fixture(scope="module")
+def snmp(ssh_appliance):
+    ssh_appliance.run_command("echo 'disableAuthorization yes' >> /etc/snmp/snmptrapd.conf")
+    ssh_appliance.run_command("systemctl start snmptrapd.service")
+    yield
+    ssh_appliance.run_command("systemctl stop snmptrapd.service")
+    ssh_appliance.run_command("sed -i '$ d' /etc/snmp/snmptrapd.conf")
```

Comment:
Consider throwing an exception to error out the test if the service can't be started, and warning if shutting it down fails. 

Consider using request fixture and introspecting a specific filename instead of dumping the file after each test.